### ISSUE TITLE
Manuscript review: Have a strategy

### ISSUE THREAD

**Issue by lukefretwell** [proudlyserving/book#230]
[Reference](https://docs.google.com/document/d/1rruJsEF8-E3qTVCv0Giw2mK43HcNS4d7233rgGk9wjw/edit?usp=sharing)

- [ ] First pass (@lukefretwell / @rebeccawoodbury)
- [ ] Editorial review (@MorganGriffin-BloomWorks)
- [ ] Address/resolve review feedback (@lukefretwell / @rebeccawoodbury)
- [ ] Send author email for resolution (if needed) (@rebeccawoodbury)
- [ ] Reconcile 'Learn more' section (@rebeccawoodbury)
- [ ] Update web page copy

**Comment by MorganGriffin-BloomWorks** [comment]
Hey! Some thoughts:

- The chapter is really abstract. _Gov. needs to be strategic, so they need a strategy, so they need a strategic mindset_. But there's never clarity about what we're talking about. 
- Like, what is strategy here? It sounds like this thing that solves all the problems, but not clear what it is or how it works. There are no specifics so nothing for a reader to grab onto. 
- Speaking of strategies and goals..."As trends progress and goals are met, strategy needs to evolve, but still must be the underlying foundation to everything you do." This is weird. A strategy is a **means to meeting a goal** isn't it? The goal is what's most important. Strategy is the tool. You need to constantly reevaluate your strategy to make sure it is working — i.e., meeting the goal. If it isn't meeting the goal, you change strategies. You don't use the same underlying strategy to meet every goal — that's the old cliche, to a hammer, everything looks like a nail. 
- The section "What happens if you don't have a strategy" makes sense as a header, but it's too vague. Just says it results in people feeling like gov doesn't care and problems with staff retention. How? Why?

Bottom line:
- Topic is important and there's some helpful stuff here, but I think it needs an extensive rewrite
- Absolutely need examples of actual goals/strategies and less abstractions
- Could also use examples to illustrate the cost of not having a strategy
